python/doc/examples/probabilistic_modeling/probabilistic_modeling.rst

```diff
@@ -45,6 +45,7 @@ Stochastic processes
     process_manipulation.ipynb
     field_manipulation.ipynb
     timeseries_manipulation.ipynb
+    processsample_manipulation.ipynb
```

Comment:
At the beginning, it is not useful to give the list of the methods illustrated.
Add real titles o the graphs by naming the objects.
The tables giving the realizations of a Process or a SampleProcess are not usefull.